src/components/stepper/StepperLabel.tsx

```diff
@@ -0,0 +1,18 @@
+import { Icon } from '../icon';
+import { Typography } from '../typography';
+import { StepperLabelProps } from './Stepper.types';
+
+import { StepperButton, StepperLabelWrapper } from './StepperLabel.styles';
+
+export const StepperLabel = ({ icon, label, variant = 'inactive', onClick, ...props }: StepperLabelProps) => {
+  return (
+    <StepperButton $variant={variant} onClick={onClick} disabled={variant === 'disabled'} {...props}>
+      <StepperLabelWrapper $variant={variant}>
+        <Icon size="1.25x" icon={icon} />
```

Comment:
Can we have icons defined by default? I mean the `circle check` and an `incomplete circle check`. In most situations we are not going to change this icons.

It is ok to support that you can add your own icons but I would define the `circle check` here

Reply:
There is a variant without icons, so I can add a new prop `showIcons` and define an icon for every variant if it is not defined from the props

Reply:
Changed, @echosergio @JaimeFLandbot what do you think?

Reply:
I was checking the design in Figma and there are some cases that are not covered:

![image](https://github.com/landbot-org/lui/assets/6536180/e9f0394a-5513-4f57-9d8b-dc520ecc9a2b)

I think we are missing some hover behaviors and some icons are not the same, what do you think?

Reply:
Resolved